rust/gui-client/docs/intended_behavior.md

```diff
@@ -41,6 +39,60 @@ Best performed on a clean VM
 1. Expect `zipinfo` to show a single directory in the root of the zip, to prevent zip bombing
 1. Expect two subdirectories in the zip, "connlib", and "app", with 3 and 2 files respectively, totalling 5 files
 
+## Smoke test checklist (Windows)
```

Comment:
Thanks again for adding these. They'll be super useful for other devs and/or writing automated tests.